src/pages/App.tsx

```diff
@@ -73,7 +73,7 @@ const HeaderWrapper = styled.div`
   justify-content: space-between;
   position: fixed;
   top: 0;
-  z-index: 2;
+  z-index: 9999;
```

Comment:
maybe we should add an enum for this value and that here?
https://github.com/Uniswap/interface/blob/622581ee0aeaa6c72aa8345fe607810278ae5e0b/src/theme/index.tsx#L55

and refactor where this 9999 is used elsewhere in the app, I think it's just a couple of others

Reply:
sounds like a good idea done